minimesos/src/main/java/com/containersol/minimesos/main/CommandPs.java

```diff
@@ -0,0 +1,60 @@
+package com.containersol.minimesos.main;
+
+import com.containersol.minimesos.MinimesosException;
+import com.containersol.minimesos.cluster.ClusterRepository;
+import com.containersol.minimesos.cluster.MesosCluster;
+import com.containersol.minimesos.mesos.MesosClusterContainersFactory;
+import com.containersol.minimesos.state.Framework;
+import com.containersol.minimesos.state.State;
+import com.containersol.minimesos.state.Task;
+
+import java.io.PrintStream;
+
+/**
+ * Lists tasks on the cluster
+ */
+public class CommandPs implements Command {
+
+    private ClusterRepository repository = new ClusterRepository();
+
+    private PrintStream output = System.out; // NOSONAR
+
+    public CommandPs(PrintStream output) {
+        this.output = output;
+    }
+
+    public CommandPs() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1186)